tests/fixtures/glue/datacontract.yaml

```diff
@@ -0,0 +1,19 @@
+dataContractSpecification: 0.9.3
+id: my-data-contract-id
+info:
+    title: My Data Contract
+    version: 0.0.1
+models:
```

Comment:
We could also write the `server` information from the used connection

Reply:
Added the Server model, including account, location and name. But Format is part of the table config in Glue. You can have different formats in the same glue database. Leaving the field empty for now.